src/components/AddUserForm/AddUserForm.js

```diff
@@ -0,0 +1,147 @@
+        };
+
+        addUser(user);
+
+        this.userNameRef.current.value = "";
+        this.userLastNameRef.current.value = "";
+        }
+    };
+
+    onReset = () => {
+        this.userNameRef.current.value = "";
+        this.userLastNameRef.current.value = "";
+        // todo: имплементнуть функцию скидывания значений, введеных пользователем
+    };
+
+    // onAlternativeSubmit = (event) => {
+    //     event.preventDefault();
+    //     const {addUser} = this.props;
+    //
+    //     const formData = new FormData(event.target);
+    //     debugger
+    //     const user = {
+    //         name: formData.get('userName'),
+    //         lastName: formData.get('userLastName'),
+    //         id: uniqId()
+    //     };
+    //     addUser(user);
+    //
+    //     console.log(user)
+    // };
+
+    render() {
+        return (
+            <form className={CN} onSubmit={this.onSubmit}>
+                <h2>Create new User</h2>
+                <div className="form-group">
+                    <label htmlFor="title" className="input-group-text">Enter post title:</label>
+                    <input
+                        ref={this.userNameRef}
+                        className={`form-control ${this.state.errorUserName  ? "error-input" : ""}`}
+                        type="text"
+                        name="userName"
+                        onChange={this.onInputChange}
+                    />
+                <div className={'error'}> {this.userNameRef.current ? this.state.errorUserName : ""}</div>
+                </div>
+                <div className="form-group">
+                    <label htmlFor="text" className="input-group-text">Enter post text:</label>
+                    <input
+                        ref={this.userLastNameRef}
+                        className={`form-control ${this.state.errorUserLastName ? "error-input": ""}`}
+                        type="text"
+                        name="userLastName"
+                        onChange={this.onInputChange}
+                    />
+                    <div className={'error'}> {this.userLastNameRef.current ? this.state.errorUserLastName : ""}</div>
+                </div>
+                <div className={'error h3'}>
+                    {this.state.currentError ? <img className={'img-warning'} src={ImgWarning} alt=""/> : ""}
+                    <br/>
+                    {this.state.currentError}
+                </div>
+
+                {/* todo: Добавить здесь <div> сообщение об ошибке, если такая произошла */}
+                {/* todo: стилизуйте это сообщение об ошибке, чтоб текст был красным и броским */}
+                {/* todo: можете добавить иконку с восклицательным знаком, чтоб привлечь внимание пользователя */}
+                {/* todo: под ошибкой имеется ввиду, что пользователь не ввел данные в инпуты и нажал submit */}
+                <Button isDisabled={!this.userNameRef.current && !this.userLastNameRef.current}
```

Comment:
This check is incorrect
this.userNameRef.current is a ref to input and you need to check if input's value is empty
Please make a fix here